docs/blog/2023/09/schema-management-disaster-recovery-in-klaw.md

```diff
@@ -0,0 +1,195 @@
+
+**Confluent** : https://docs.confluent.io/platform/current/schema-registry/develop/api.html#schemas
+
+## Schema management in Klaw
+
+In Klaw, each Kafka topic is associated with a specific owner, usually a designated team. The owner team is
+automatically granted the role of schema owner, giving them the authority to create new schemas and oversee their
+evolution.
+
+Klaw ensures data consistency, reliability, and secure access, aligning with organizational policies. It also controls
+schema registration, permitting only the designated owner to manage schemas and preventing unauthorized modifications
+by other teams.
+
+
+### Request for a Schema in Klaw
+
+After creating a topic, the designated team that owns the topic gains the exclusive right to register a schema for it.
+As the topic owner, you have the authority to approve, initiate, or reject any schema-related requests for that topic.
+
+
+![image](../../../static/images/blogimages/SchemaRequest-Workflow.png)
+
+For more information on registering a schema **[see here](../../../docs/HowTo/schemas/Request-a-schema)**
+
+Watch the video below for a step-by-step demonstration of registering a schema in Klaw.
+
+<iframe src="https://drive.google.com/file/d/1g_wngVYcSgtGIWjwMxNT0ck--zGIKuew/preview" width="640" 
+height="480" allow="autoplay"></iframe>
+
+### Schema migration from Schema registry to Klaw
+
+Backing up schemas from the schema registry is essential for ensuring recovery in the event of server failures or
+unexpected disasters. While schemas are typically stored in a Kafka topic with replication, there may be cases where
+retrieving them is difficult or impossible.
+
+Klaw simplifies this by connecting directly to the schema registry server, retrieving all available schemas associated
+with topics, and integrating them into its metastore. This backup process is straightforward and takes only a few
+clicks, regardless of the volume of schemas or topics.
+
+
+For more information on synchronizing schemas from the schema registry to Klaw, 
+**[refer to](../../../docs/HowTo/kafka-cluster-migration/sync-schemas-from-cluster)**
+
+The following video demonstrates how schemas are synchronized to Klaw for a selection of topics already present in
+Karapace.
+
+
+<iframe src="https://drive.google.com/file/d/1iSkAUd7jVD7Zt6OH_-Gc2NGyIbpXi2VK/preview" width="640" height="480"
+allow="autoplay"></iframe>
+
+### Schema migration from Klaw to Schema Registry
+
+After successfully backing up schemas in Klaw, you have the flexibility to register them either to a new schema registry
+cluster or integrate them into an existing one.
+
+Select the schemas you want to sync with the cluster. Once you submit your choices, Klaw will register the selected
+schemas directly into your designated schema registry environment.
+
+
+For more information on this synchronization process, 
+**[refer to](../../../docs/HowTo/kafka-cluster-migration/sync-schemas-to-cluster)**
+
```

Comment:
these links dont look like links it just looks like the statement is bolded and then ends
![image](https://github.com/Aiven-Open/klaw-docs/assets/121855584/b6681005-4d60-468f-a54d-bf5679de728f)

Reply:
Updated text, but it is not possible to underline text or apply colors. To get underlines, need to install a different plugin I see.